fuzz/src/test/java/com/dylibso/chicory/fuzz/TestModule.java

```diff
@@ -0,0 +1,111 @@
+package com.dylibso.chicory.fuzz;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.dylibso.chicory.log.Logger;
+import com.dylibso.chicory.log.SystemLogger;
+import com.dylibso.chicory.runtime.Instance;
+import com.dylibso.chicory.runtime.Module;
+import com.dylibso.chicory.wasm.types.FunctionType;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class TestModule {
+    private static final Logger logger = new SystemLogger();
+    WasmTimeWrapper wasmtime = new WasmTimeWrapper();
+    ChicoryCliWrapper chicoryCli = new ChicoryCliWrapper();
+
+    public List<String> paramsList(FunctionType type) {
+        return Arrays.stream(type.params()).map(p -> randomNumber()).collect(Collectors.toList());
+    }
+
+    public String randomNumber() {
+        // TODO: 2 digits integer seems not enough, but a starting point ...
+        return RandomStringUtils.randomNumeric(2);
```

Comment:
This is a good start and not worth worrying too much about right now, but i think generally you don't want a uniform random distribution. There are some values which are more likely to break things than others. You want to try those first. There are probably some libraries that can help us define these. It's probably worth asking an expert for guidance on this part.

Reply:
we may also need to "reduce" inputs after a failure. minimizing a broken example down to the smallest failure case. again i assume there are some libraries in java that can help with this.

Reply:
maybe a library like this could help https://github.com/pholser/junit-quickcheck

Reply:
Awesome idea!
I'm not 100% sure how to properly integrate it, and I think we should gain some experience with the tools themselves but surely something we can explore at some point.